src/swagger/subscribe/add.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * @swagger
+ * /subscribers:
+ *   post:
+ *     tags:
+ *       - Subscribers
+ *     summary: Add a new subscriber
+ *     description: Endpoint to add a new subscriber to the mailing list.
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             type: object
+ *             properties:
+ *               email:
+ *                 type: string
+ *                 format: email
+ *                 description: The subscriber's email address.
+ *             required:
+ *               - email
+ *             example:
+ *               email: user@example.com
+ *     responses:
+ *       201:
+ *         description: Subscriber added successfully.
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 status:
+ *                   type: string
+ *                   example: success
+ *                 message:
+ *                   type: string
+ *                   example: Subscriber saved successfully
+ *                 result:
+ *                   $ref: '#/components/schemas/Subscriber'
```

Comment:
result is shown as "string" in the swagger UI which is incorrect

Reply:
with the components defined, there shouldn't be any problem.